### PR TITLE
Ultenic K10 fixes

### PR DESCRIPTION
**Pull request by blakadder**
Fixes for Ultenic K10 air fryer config

### CODE REVIEW

File: custom_components/tuya_local/devices/ultenic_air_fryer.yaml [hunk 1]
```diff
@@ -181,30 +184,47 @@ secondary_entities:
...
+        type: boolean
+  - entity: binary_sensor
+    name: Shake
+    icon: "mdi:car-brake-alert"
+    dps:
+      - id: 104
+        name: sensor
+        type: string
+        mapping:
+          - dps_val: 0
```
Comment (make-all) [inline custom_components/tuya_local/devices/ultenic_air_fryer.yaml:221]: numeric values of string type need double quotes.

File: custom_components/tuya_local/devices/ultenic_air_fryer.yaml [hunk 2]
```diff
@@ -86,25 +85,25 @@ secondary_entities:
         name: sensor
         mapping:
           - dps_val: "0"
-            value: Standby
+            value: standby
```
Comment (make-all) [inline custom_components/tuya_local/devices/ultenic_air_fryer.yaml:88]: These were capitalised as there is no translation for sensor values, so they will appear in the UI as-is.

File: custom_components/tuya_local/devices/ultenic_air_fryer.yaml [hunk 3]
```diff
@@ -122,19 +121,18 @@ secondary_entities:
...
         mapping:
-          - dps_val: "0"
+          - dps_val: c
             value: C
-          - dps_val: "1"
+          - dps_val: f
             value: F
   - entity: sensor
-    name: Temperature
+    name: Internal Temperature
```
Comment (make-all) [inline custom_components/tuya_local/devices/ultenic_air_fryer.yaml:129]: HA naming convention is that only the first word in a name gets capitalised, so if changing this the t also needs changing.

**Comment by blakadder** [comment]
thanks for the review and comments, those are helpful in making future PRs better

**Pull request merged**